core/trino-main/src/main/java/io/trino/SystemSessionProperties.java

```diff
@@ -187,6 +187,10 @@ public final class SystemSessionProperties
     public static final String FAULT_TOLERANT_EXECUTION_MIN_PARTITION_COUNT = "fault_tolerant_execution_min_partition_count";
     public static final String FAULT_TOLERANT_EXECUTION_MIN_PARTITION_COUNT_FOR_WRITE = "fault_tolerant_execution_min_partition_count_for_write";
     public static final String FAULT_TOLERANT_EXECUTION_MIN_SOURCE_STAGE_PROGRESS = "fault_tolerant_execution_min_source_stage_progress";
+    private static final String FAULT_TOLERANT_EXECUTION_SMALL_STAGE_ESTIMATION_ENABLED = "fault_tolerant_execution_small_stage_estimation_enabled";
+    private static final String FAULT_TOLERANT_EXECUTION_SMALL_STAGE_ESTIMATION_THRESHOLD = "fault_tolerant_execution_small_stage_estimation_threshold";
+    private static final String FAULT_TOLERANT_EXECUTION_SMALL_STAGE_SOURCE_SIZE_MULTIPLIER = "fault_tolerant_execution_small_stage_source_size_multiplier";
```

Comment:
fault_tolerant_execution_small_stage_source_size_estimation_multiplier

Reply:
no... :)
It does not parse it is not `estimation multiplier` but `size multiplier` 
If so I would make it `fault_tolerant_execution_small_stage_estimation_source_size_multiplier` - but I am not convinced.

Reply:
`fault_tolerant_execution_small_stage_estimation_source_size_multiplier` feels better😊But I won't insist